packages/dotcom-ui-header/browser.js

```diff
@@ -16,7 +17,29 @@ export const init = (headerOptions = {}) => {
     '.o-header [data-n-topic-search], .o-header__drawer [data-n-topic-search]'
   )
   topicSearchElements.forEach((element) => {
-    new TopicSearch(element, headerOptions)
+    const oheaderContainer = element.parentElement
+    const oheaderContainerParent = oheaderContainer?.parentElement
+    const isDrawer = oheaderContainerParent?.parentElement?.getAttribute('data-o-header-drawer') === 'true'
+
+    const form = oheaderContainer?.querySelector('form')
+    const input = form?.querySelector('input')
+    const typeaheadContainer = document.createElement('div')
+    typeaheadContainer.id = `suggestions-${input?.id}`
+    typeaheadContainer.className = 'typeahead__main-container'
+    typeaheadContainer.role = 'listbox'
+
+    if (!form || !input) return
```

Comment:
!form should be inside the else?

Reply:
Hm... I added this as the precaution. It's should never happen in production. The form should always be present, however if input of form doesn't exist there is no reason to include or initiate the typeahead.

So it's okay to catch it early before anything else